ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -1159,6 +1159,54 @@ def test_tensorflow_realdiv(
         y=x[1],
     )
 
+    
+# tile
+@st.composite
+def _multiple_shape_helper(draw):
+    input_dtype, input_array, input_shape = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("numeric"),
```

Comment:
Would this work for all `"valid"` data types? It seems to me that we should be able to repeat booleans too for example.